### PR TITLE
stacks: add transaction and tranfer builders

### PR DESCRIPTION
**Pull request by fdmirza**
This PR add implementation of transaction builder and transfer builder.
This should be merged after https://github.com/BitGo/BitGoJS/pull/1050

cc @wesgraham

### CODE REVIEW

File: modules/account-lib/package.json [hunk 1]
```diff
@@ -39,9 +39,12 @@
     "@hashgraph/sdk": "^1.2.1",
     "@stablelib/hex": "^1.0.0",
     "@stablelib/sha384": "^1.0.0",
+    "@stacks/keychain": "^1.2.2",
+    "@stacks/transactions": "^1.2.2",
     "@taquito/local-forging": "6.3.5-beta.0",
     "@taquito/signer": "6.3.5-beta.0",
     "@types/lodash": "^4.14.151",
+    "@stacks/transactions": "^1.2.2",
```
Comment (tylerlevine) [inline modules/account-lib/package.json:47]: How come this appears twice in the deps list?

**Review by tylerlevine — COMMENTED** [review]
I didn't do a full review on this but one thing did catch my eye. Please re-request me if you'd like a more thorough review

File: modules/statics/src/base.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export enum CoinFamily {
   OFC = 'ofc',
   RMG = 'rmg',
   RBTC = 'rbtc',
+  STACKS = 'stacks',
```
Comment (argjv) [inline modules/statics/src/base.ts:36]: Isn't the ticker STX?

Comment (fariedt) [reply modules/statics/src/base.ts:36]: I'll add that change to #1050 in a bit.

**Comment by fariedt** [comment]
This PR is supposed to build on top of #1050, but there appear to be a few missing commits from that branch.  @fdmirza is out of the office today, and should be able to look into it tomorrow.

File: modules/account-lib/src/coin/stacks/keyPair.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { publicKeyToAddress } from '@stacks/encryption';
```
Comment (wesgraham) [inline modules/account-lib/src/coin/stacks/keyPair.ts:1]: looks like this needs to be rebased/refactored based on key generation PR

File: modules/account-lib/src/coin/stacks/transaction.ts [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+  private _stxTransaction: StacksTransaction;
+  protected _type: TransactionType;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  /** @inheritdoc */
+  canSign(key: BaseKey): boolean {
```
Comment (wesgraham) [inline modules/account-lib/src/coin/stacks/transaction.ts:29]: Note: can we add a todo here if we aren't implementing?

**Pull request closed without merging**